.travis.yml

```diff
@@ -22,3 +22,6 @@ deploy:
     branch: master
     tags: true
     condition: $TOXENV == py27
+addons:
+  hosts:
+    - ip6-localhost
```

Comment:
That is a good question. The only thing I found when searching was [this](https://docs.travis-ci.com/user/hosts/)